src/libtriton/api/api.cpp

```diff
@@ -929,7 +929,20 @@ namespace triton {
     return this->symbolic->isOptimizationEnabled(opti);
   }
 
+  bool API::isMemorySymbolic(const triton::arch::MemoryAccess& mem) const{
+	  this->checkSymbolic();
+	  triton::usize symbId = triton::api.getSymbolicMemoryId(mem.getAddress());
+	  triton::engines::symbolic::SymbolicExpression* symbExp = triton::api.getSymbolicExpressionFromId(symbId);
+	  return symbExp->isSymbolized();
+  }
 
+  bool API::isRegisterSymbolic(const triton::arch::Register& reg) const{
+	  this->checkSymbolic();
+	  triton::usize symbId = triton::api.getSymbolicRegisterId(reg);
+	  triton::engines::symbolic::SymbolicExpression* symbExp = triton::api.getSymbolicExpressionFromId(symbId);
```

Comment:
Same as the previous function. You have to manage the case where `symbId` is `UNSET` and use `this->` instead of `triton::api`.